data/coalition/resistance intro.txt

```diff
@@ -0,0 +1,1791 @@
+			`	It thinks for a moment, then answers, "No. We never intended to fight. It was the Saryds, Kimek and Arachi of old who fought us. We are not a warring people."`
+			`	Seemingly satisfied with the answer, Oobat moves on to the next. She questions the Quarg for about ten minutes on their activities outside the Coalition and their relation with the humans and other species. She even has you move the camera around for a bit to show the local human-Quarg interactions, as the Quarg compares dealing with humanity to dealing with the Coalition species. Oobat moves from question from question as if in a checklist, until she arrives at the last one. "About finished, we are. Thankful for your answers I am. Just one last thing, to appease the people back home. Interested in returning to take back the rings, are you?"`
+			`	"Of course," it says without hesitation. "The rings are our homes. My home." Clearly not expecting that answer, Oobat looks at you in confusion.`
+			choice
+				`	"It's been thousands of years. You're still lingering on those rings?"`
+				`	"If you want them back, why haven't you returned by now?"`
+					goto why
+			`	"Lingering on... 'those' rings? Our rings. Our homes," it replies. "Do you so easily forget yours?" The Quarg crosses one arm, grabbing its shoulder, lowering its head as its back curves. Patches of skin start turning pitch black, and it brings its other arm to its head as if in pain.`
+				goto accept
+			label why
+			`	"Because that would have forced a fight. The three... these Heliarchs, would not listen. They never listened," it answers, crossing one arm and grabbing its shoulder, lowering its head as its back curves. Patches of skin start turning pitch black, and it brings its other arm to its head as if in pain.`
+			label accept
+			`	By now the atmosphere's changed, with no humans in sight around you. Dozens more Quarg begin to form circles around your ship, eyes fixated on Oobat. The ones closest to you watch the one who was being interviewed carefully, like waiting for some sign, as its whole body shifts color. Oobat is still looking at you, confusion and worry written on her face.`
+			choice
+				`	"I think that does it for the interview."`
+				`	"We should leave. Now."`
+			`	She nods, and gives a quick thank to the Quarg again, turning to go back inside your ship. The Quarg stops her though, reaching out one arm quickly to grab one of her legs. Slowly, its skin turns back to the grayish blue from before, as it calms down. "Should you succeed, please, permit my return home." The Quarg around you disperse, and the one who was interviewed stands up again and moves back to other sections of the station.`
```

Comment:
The Quarg never gave me the impression they were capable of speed, or that they'd have the strength to actually stop an Arach by grabbing a leg. This might be better restructured as a spoken interruption - or better yet, have them raise the issue of desiring a statement before you turn to leave very promptly.

Reply:
First Contact: Kuwaru Efreti
`In your past experience, you've never seen the Quarg move with anything approaching haste or excitement, but when you land on this station a group of them run up to you, moving faster than you thought they were able to.`

It's more that Oobat was shocked by the sudden grab rather than the Quarg having the strength to hold her in place.